README.md

```diff
@@ -38,4 +38,6 @@ Usage: run_log4j_patcher.sh (subcommand) [options]
         SKIP_HDFS         If non-empty, skips scanning and patching .tar.gz files in HDFS
         RUN_SCAN          If non-empty, runs a final scan for missed vulnerable files. This can take several hours.
 
-HDP Notes : Currently the HDP removal scrips works on folder `/user/`  on HDFS. Please modify/extent in The `hdp_log4j_jndi_removal.sh` around `line 19`.  
\ No newline at end of file
+HDP Notes : Currently the HDP removal scrips works on folder `/user/`  on HDFS. Please modify/extent in The `hdp_log4j_jndi_removal.sh` around `line 19`.  
+
+CDP Notes : By default, the script will scan all parcel directories. Therefore, it is advisable to deactivate and remove all obsolete/unused parcels.
```

Comment:
Great point !
Furthermore, the script should get an option to allow skipping those huge parcel .tar.gz files alltogether  because:
- AFAIU those parcels are not required to be patched for the running cluster anyway 
- The 1.customer I ran the scripts, I had some major trouble with "only" 10GB /tmp space on the CDSW (master)node , which was of course not enough  for the patcher script  to handle the 5+ GB sized tar.gz (since it needs >2x the space for un the unpacked version ) .. on top of that, the .tar.gz backups were somehow written to a dir in /tmp (seems like a bug, only in tar.gz function .. in the jar one it worked properly into the default backup dir)
  - I also suggest a note in the README about the required /tmp  free space
  -  even better, the script should have an option to choose a different (base)tmp-dir than /tmp

 I am wondering

Reply:
@lhoss Regarding the /tmp issue: We have internally patched all calls to mktemp with the -p option.

Reply:
Would be helpful to merge if you're OK with it.  Please make `/tmp` a separate issue/PR

Reply:
Yes OK to merge and I should create a sep issue to have a configurable /tmp folder!
@fetzerms a PR of your internal featzre would be great ( and I'm happy to use/test it tomorrow on a customer Dev cluster)